src/main/fc/fc_msp.c

```diff
@@ -822,6 +823,20 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         }
         break;
 
+#ifdef VTX_SMARTAUDIO
```

Comment:
I think this was done on purpose, waiting for @jflyper who told us he would need some more time consolidate the different VTX interfaces (incl. upcoming Tramp). So please keep in mind that this is a temporary solution, that might just stay as a PR, so that people needing it will just compile there own version.